### PR TITLE
Add download-old-gf-release target to fetch the 0.80/0.83 GF release

### PR DESCRIPTION
**Pull request by madig**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -70,3 +71,6 @@ release: $(TTF) $(MONO_TTF) LICENCE.txt
 	$(CP) $(BUILD_DIR)/UbuntuMono-B.ttf $(RELEASE_DIR)/UbuntuMono-Bold.ttf
 	$(CP) $(BUILD_DIR)/UbuntuMono-BI.ttf $(RELEASE_DIR)/UbuntuMono-BoldItalic.ttf
 	$(CP) LICENCE.txt $(RELEASE_DIR)/LICENSE.txt
+
+download-old-gf-release:
+	sh tools/download-old-gf-release.sh
```
Comment (sladen) [inline Makefile]:
```sh
wget 'https://fonts.google.com/download?family=Ubuntu'
```

**Comment by sladen** [comment]
```sh
wget https://fonts.google.com/download?family=Ubuntu
```
(returns `Ubuntu.zip`)

**Comment by madig** [comment]
That will presumably download the most recent one from GF. Do we always want that one or the current 0.80/0.83 one?

**Comment by sladen** [comment]
Probably the $most_recent.  It's also probably neater and more reliable than the hashes.

(It was suggested in the bug to also begin a repository of released versions; which can perhaps do once things are neatly tagged in source form).

**Pull request merged**